modules/nw-egress-ips-config-object.adoc

```diff
@@ -28,8 +31,5 @@ spec:
         routingViaHost: false
       genevePort: 6081
 ----
-<1> The `egressIPConfig` holds the configurations for the options of the `EgressIP` object. Changing these configurations allows you to extend the `EgressIP` object.
-
-<2> The value for `reachabilityTotalTimeoutSeconds` accepts integer values from `0` to `60`. A value of 0 disables the reachability check of the egressIP node. Values of `1` to `60` correspond to the duration in seconds between probes sending the reachability check for the node.
-
-
+<1> The `egressIPConfig` holds the configurations for the options of the `EgressIP` object. By changing these configurations, you can extend the `EgressIP` object (in what way?).
+<2> The value for `reachabilityTotalTimeoutSeconds` accepts integer values from `0` to `60`. A value of `0` disables the reachability check of the egressIP node. Values of `1` to `60` correspond to the duration in seconds between probes sending the reachability check for the node.
```

Comment:
Does:

> Values of `1` to `60` correspond to the duration in seconds between probes sending the reachability check for the node.

Those this sentence indicate the "interval" situation as mentioned in the Jira ticket? https://issues.redhat.com/browse/OCPBUGS-38002